src/write/elf/object.rs

```diff
@@ -279,6 +280,7 @@ impl<'a> Object<'a> {
             Architecture::Riscv64 => elf::EM_RISCV,
             Architecture::S390x => elf::EM_S390,
             Architecture::Sparc64 => elf::EM_SPARCV9,
+            Architecture::LoongArch64 => elf::EM_LOONGARCH,
```

Comment:
Please move this before `Architecture::Mips`.

Reply:
Done.